suse2013/fo/xref.xsl

```diff
@@ -184,13 +217,24 @@
         select="@linkend"/>' points to a chapter (id='<xsl:value-of
           select="$target.chapandapp/@id"/>') with a different language than the main book.</xsl:message>
       </xsl:if>
-
+      <xsl:choose>
```

Comment:
Maybe instead of having a choose here, create a new xsl:param which sets the text color using a similar choose?